enews/enews-node/enews-fetcher.js

```diff
@@ -70,8 +70,8 @@ function EnewsFetcher(token) {
         var attachment = message.attachments[0];
         return {
             originalMessage: message.text,
-            title: attachment.title,
-            link: attachment.title_link,
+            title: attachment.title ? attachment.title : (attachment.text ? attachment.text : ''),
```

Comment:
It would be more readable with `attachment.title || attachment.text || ''` which reads like "title or text or empty string".
Same for the following line.